pkg/traceql/lexer.go

```diff
@@ -245,7 +247,7 @@ func isAttributeRune(r rune) bool {
 	}
 
 	switch r {
-	case scanner.EOF, '{', '}', '(', ')', '=', '~', '!', '<', '>', '&', '|', '^':
+	case scanner.EOF, '{', '}', '(', ')', '=', '~', '!', '<', '>', '&', '|', '^', ',':
```

Comment:
Highlighting that this means we can't use attribute names containing a comma after this change.  I think that is ok, we will figure out escaping at some point.